src/sidebar/test/group-fixtures.js

```diff
@@ -0,0 +1,172 @@
+'use strict';
+
+const Chance = require('chance');
+const chance = new Chance();
+
+function group () {
+  const id = chance.hash({length: 15});
+  const name = chance.string();
+  const group = {
+    id: id,
+    name: name,
+    links: {
+      html: `http://localhost:5000/groups/${id}/${name}`,
+    },
+    type: 'private',
+  };
+  return group;
+}
+
+function organization (options={}) {
+  const org = {
+    id: chance.hash({length : 15}),
+    name: chance.string(),
+    logo: chance.url(),
+  };
+  return Object.assign(org, options);
+}
+
+function defaultOrganization () {
+  return {
+    id: '__default__',
+    name: 'Hypothesis',
+    logo: 'http://example.com/hylogo',
+  };
+}
+
+function expandedGroup (options={}) {
+  const expanded = group();
+  expanded.organization = organization();
+
+  return Object.assign(expanded, options);
+}
+
+function expandedOrganizations () {
```

Comment:
Any reason not to use `expandedGroup` to generate `expandedOrganizations`? This will make it easier to keep in sync with any future API changes. The name confused me as well, since the result is a list of groups _with organizations expanded_.

Reply:
After looking through the rest of the files, it seems that this function is not used in this PR. Is it still needed?

Reply:
Nope! Good catch. The fixture stuff evolved as I was going...classic stuff...